addon-test-support/index.ts

```diff
@@ -2,6 +2,7 @@ export { default as a11yAudit } from './audit';
 export { default as a11yAuditIf } from './audit-if';
 export { setRunOptions, getRunOptions } from './run-options';
 export { setEnableA11yAudit, shouldForceAudit } from './should-force-audit';
+export { shouldUseMiddlewareReporter } from './should-use-middleware-reporter';
```

Comment:
What do you think about renaming this to `useMiddlewareReporter` to keep it more terse?

Reply:
Yeah, that reads much better. I'll update.